### ISSUE TITLE
The project does not install due to github's dropping of the git:// protocol support

### ISSUE THREAD

**Issue by d-ph** [pusher/pusher-js#568]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
`npm install pusher/pusher-js` does not complete.
```
$ npm install pusher/pusher-js                                                                                                                                                                       [10:06:31]
npm ERR! code 1
npm ERR! Command failed: git submodule update -q --init --recursive
npm ERR! warning: templates not found in /var/folders/pb/cwq_px3d3xjdlr931f9h_bqhsdlkx_/T/pacote-git-template-tmp/git-clone-f7be7ad6
npm ERR! fatal: remote error: 
npm ERR!   The unauthenticated git protocol on port 9418 is no longer supported.
npm ERR! Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```

**If the current behavior is a bug, please provide the steps to reproduce and
1. Create a new directory. 
2. `npm init`
3. `npm install pusher/pusher-js`

**What is the expected behavior?**
It installs.

**Which versions of Pusher, and which browsers / OS are affected by this issue?
All.

** More
Github ceased to support the git:// protocol, which this project uses in its .gitmodules at [link](https://github.com/pusher/pusher-js/blob/6b785e77672802174f77bb23f2ec239e1ff7a35c/.gitmodules#L3). Changing the protocol there to `https://` should fix the problem.

**Comment by bognerf** [comment]
Same here, can't run `npm install` any longer without hitting this bug.

**Comment by benw-pusher** [comment]
Thanks for reporting.  I've followed the replication instructions and I don't encounter the error. I'll continue to investigate.

**Comment by d-ph** [comment]
Hi Ben,

You're right, the issue cannot be reproduced anymore. I suppose github reverted the change that they deployed 2 days ago. It's most likely best to assume that they will re-attempt to deploy it again, so I recommend that the protocol in the .gitmodules is changed from git:// to https:// anyway. Please see the blog post that they mentioned in the error message: https://github.blog/2021-09-01-improving-git-protocol-security-github/ They say that they want all the changes to be permanently deployed by March 15 this year. 

I'll leave this ticket open in case there are further questions from you. If there's nothing more to do, then please feel free to close it.

Thanks.

**Comment by benw-pusher** [comment]
Thanks @d-ph. Reading the blog post you have shared it does seem like the change you mention is a good idea. I'll share this issue with the engineering team internally to review and update. 

Ben

**Comment by benw-pusher** [comment]
Keeping open.

**Comment by benw-pusher** [comment]
Preventing closure

**Comment by stof** [comment]
should be closed as #587 has been merged
